app/js/redux/selectors.js

```diff
@@ -5,6 +5,12 @@ export const selectPlayers = state => state.players;
 export const selectTricks = state => state.tricks;
 export const selectDeck = state => state.deck;
 
+export const selectNbPlayers = createSelector(
```

Comment:
Why not, après il s'agit d'un calcul très simple à réaliser, et définir un sélecteur consomme de la mémoire donc il ne faut pas forcément en créer systématiquement.